vendor/processors/authorizenet.py

```diff
@@ -209,7 +210,10 @@ def create_billing_address(self, api_address_type):
         billing_address.firstName = " ".join(self.payment_info.data.get('full_name', "").split(" ")[:-1])[:50]
         billing_address.lastName = (self.payment_info.data.get('full_name', "").split(" ")[-1])[:50]
         billing_address.company = self.billing_address.data.get('company', "")[:50]
-        billing_address.address = ", ".join([self.billing_address.data.get('address_1',""), str(self.billing_address.data.get('address_2', ""))])[:60]
+        address_lines = self.billing_address.data.get('address_1', "")
+        if self.billing_address.data['address_2']:
```

Comment:
is there a chance this might cause an error if thing is entered in address_2? should it be self.billing_address.data.get("address_2", ""), like the previous line?

Reply:
The thing is that it is setting it as None, so it returns None instead of the empty string. I did try that jejeje.